ui/app/helpers/mountable-secret-engines.js

```diff
@@ -9,7 +9,7 @@ const ENTERPRISE_SECRET_ENGINES = [
   {
     displayName: 'KMIP',
     type: 'kmip',
-    engineRoute: 'kmip.scopes',
+    engineRoute: 'kmip.scopes.index',
```

Comment:
IIRC the test failures happening now are from when I tried adding this previously - but I could be wrong!

Reply:
🤔 strange -- maybe we were testing the wrong thing? There doesn't seem to be a `kmip.scopes.js` route though